OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -835,6 +889,25 @@
 !endif
   OvmfPkg/VirtioNetDxe/VirtioNet.inf
 
+
+!if $(DASHARO_SYSTEM_FEATURES_ENABLE) == TRUE
+  DasharoModulePkg/DasharoBootPolicies/DasharoBootPolicies.inf
+!endif
+
+
+  # Hash2
+  #
+  SecurityPkg/Hash2DxeCrypto/Hash2DxeCrypto.inf {
+    <LibraryClasses>
+    BaseCryptLib|CryptoPkg/Library/BaseCryptLib/BaseCryptLib.inf
+  }
+
+  #
+  # PKCS7 Verification
+  #
+  SecurityPkg/Pkcs7Verify/Pkcs7VerifyDxe/Pkcs7VerifyDxe.inf
+
+

```

Comment:
I don;t see SmbiosMeasurementDxe anywhere in DSC and FDF but is present in UEFIPayloadPkg